stock_batch_picking/wizard/batch_picking_creator.py

```diff
@@ -13,6 +13,25 @@ class StockBatchPickingCreator(models.TransientModel):
     _name = 'stock.batch.picking.creator'
     _description = 'Batch Picking Creator'
 
+    def _default_picker_id(self):
```

Comment:
Please put this below field declarations per [model attribute ordering in Contrib docs](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field)